lucene/directory-provider/src/test/java/org/infinispan/hibernate/search/TwoNodesTest.java

```diff
@@ -60,6 +60,7 @@ public void testSomething() {
       } finally {
          nodeC.close();
       }
+      waitMembersCount(nodea, SimpleEmail.class, 2);
```

Comment:
I am personally fine with the fix.  However I am thinking we should change the waitMembersCount method to pass in a time out or something though.  It seems like this method could loop forever in a bad case :-1: